samples/xmlserializergenerator-instructions.md

```diff
@@ -6,8 +6,10 @@ You can start using the tool today following the instructions below.
 
 ## Prerequisites
 
-* [.Net Core SDK of version 2.0.2 or later](https://www.microsoft.com/net/download/windows)
-* [.NET Core runtime 2.0.3](https://github.com/dotnet/core/blob/master/release-notes/download-archives/2.0.3.md)
+The following is required for Microsoft.XmlSerializer.Generator to work. You can use command `dotnet --info` to check which versions of .NET Core SDK and runtime you may already have installed.
+
+* [.Net Core SDK 2.0.2 or later](https://www.microsoft.com/net/download/windows)
+* [.NET Core runtime 2.0.3 or later](https://github.com/dotnet/core/blob/master/release-notes/download-archives/2.0.3.md)
```

Comment:
Just noticed, we had different casing of ".NET" above.

Reply:
Good catch. I'll fix it.

Reply:
Changed all ".Net" to ".NET" throughout the doc.